### ISSUE TITLE
[ICC] [Deathbringer Saurfang] Blood Power Generation

### ISSUE THREAD

**Issue by Hbane12** [dalaranwow/dalaran-wow#4770]
[//]: # (INFORMATION ABOUT TAGS)
[//]: # (The black and yellow tag is unique. There can only be one label assigned to them - one black and one yellow)
[//]: # (Priority - ... tags are assigned by team members)
[//]: # (Your issue will be reviewed by testers and the developers will fix it in the future.)
[//]: # (Only once the issue got the label [Fixed - On Live Server] you can find the fix on your Gameserver.)
[//]: # (You can increase the relevance of your issue by adding a thumb-up emote http://i.imgur.com/RFouVVi.png)

[//]: # (***************************************************************)
[//]: # (** !!!!!!!!!!!!!DON'T DELETE THIS TEMPLATE ELSE YOUR ISSUE WILL BE CLOSED!!!!!!!!!!!! **)
[//]: # (***************************************************************)

**Current behaviour**: Tell us what happens in detail (Where, When, What, Why, Who)

On the Deathbringer Saurfang fight, Mark of the Fallen Champion is generating blood power when the person afflicted by the Mark takes damage. Also, effects like "Power Word: Shield" and "Divine Intervention" are preventing blood power from being generated by any of Deathbringer Saurfang's abilities. 

**Expected behaviour**: Tell us what should happen instead

While this is the way that the fight was first introduced, in patch 3.3.2 it was changed. Here is a link for undocumented changes to Saurfang in 3.3.2: http://wowwiki.wikia.com/wiki/Patch_3.3.2_(undocumented_changes).

The Marks of the Fallen Champion should not generate blood power, and effects which prevent damage should not be preventing blood power generation.

**Steps to reproduce the problem**:

1. Do the fight.
2. Get Mark of the Fallen Champion/Put a shield on someone with boiling blood.
3. Watch as the Mark generates ungodly amounts of blood power and shield prevents blood power generation.

**Include proofs for this behaviour**

You can research this on several pages like http://wowhead.com or http://wowwiki.wikia.com or http://www.youtube.com/
Issues without proofs will be closed and tag On Hold will be added until the proof is provided.

As listed above, the link for the proof of changes to Saurfang's mechanics is here: http://wowwiki.wikia.com/wiki/Patch_3.3.2_(undocumented_changes)

The relevant paragraph is this: "Deathbringer Saurfang will no longer gain blood power from Mark of the Fallen Champion. In addition, mitigation abilities such as Power Word: Shield will no longer prevent blood power generation."

**Include the ID for the game objects, npcs (creatures,pets,minions), spells, items, quests, instances, zones, achievements, skills**

You can research this on several pages like https://wowgaming.altervista.org/aowow/ or https://wowhead.com/

**Include Screenshots from the issue if necessary**

**Comment by Rushor** [comment]
Scripts/IcecrownCitadel: Deathbringer Saurfang - No longer gain blood power from Mark of the Fallen Champion

* In addition, mitigation abilities such as Power Word: Shield or any other proc with PROC_EX_ABSORB will no longer prevent blood power generation.
* closes: https://github.com/dalaranwow/dalaran-wow/issues/4770
* commit: c1ba97051e471bbab7a22ac470337cb79b628e03

The issue was fixed by a developer and will be tested on the Public Test Realm in the future (PTR). The changes/fixes will be applied with the NEXT changelog on the liverealm in the future.
This means that the issue exists in game at the moment i wrote this message.
**Only once the issue was marked with "Fixed - On Live Server", you can find the fix on your game server.**
Please watch http://www.dalaran-wow.com/news/ to see when the changelog and the fix was applied.

**Comment by Coztume** [comment]
Whilst that is true, thought ICC is meant to be pre-nerf here?

**Comment by Hbane12** [comment]
Eh, I mean while you could call it a nerf I think it's more of a balancing thing. The way it is now disc priests/divine intervention break the fight. At least after this change the blood power generation will be steady

**Comment by plznoz** [comment]
This is gonna make heroic too easy.

**Comment by Horvic** [comment]
Although "Blizzlike" this will remove all challenge from the encounter in both difficulties, the encounter is fine as it is.

**Comment by Rushor** [comment]
maybe discuss this in the forum as well in a new topic please. Thank you.

**Comment by Edzero** [comment]
@Rushor Lothloryen specified during the beta that this pre-nerf status was intentional.

**Comment by Rushor** [comment]
Not sure if the fix is still needed, but it's on PTR soon for further testing

**Comment by depressedme** [comment]
I just saw this post, while I was here to make my own:

Is it intentional, that you can't use Divine Intervention on targets affected by the "Mark of the Fallen Champion", whilst claiming its "breaking" the fight, it was still available for retail. Removing the abbillity will make it harder for weaker guilds, while it wont affect guilds, which is already geared for it.

And saying its removing the challenge, by using class mechanics smart is also rather confusing for me.

**Comment by Lothloryen** [comment]
@depressedme Thank you for your report. Could you provide a proof where this tactic can be seen being used on retail, please?

**Comment by JohnADC** [comment]
https://www.youtube.com/watch?v=E1icgs-5NkY

I belive this video is proof enough?

**Comment by Crispybuilt** [comment]
**Point 1:**
Unaffected by invulnerability
Cannot be reflected
All spell effects are harmful
Can only target the player
Cannot miss
Persists through death

**Point 2:**
It is considered as a major exploit because of the following reasons:
- Using DI on a Mark target completely neglets its damage to the person, hence that player does not have to be healed at all.
- Using DI on a Mark target does NOT generate Blood Power for the Saurfang (which is to be intended as this is a pre-nerf version of Saurfang)

So not only you completely break the fight with it, but you reduce the chance of wiping by a lot, just because Saurfang won't be able to heal anymore from the Mark if that person dies, which is absurd.

Why would the mechanic Mark of the Fallen Champion be invented in the first place then if it can be by-passed by a simple Divine Intervention on a target. And what exactly is Divine Intervention but a invulnerability?

It doesn't have any sense and it breaks the logic of it.

**Comment by JohnADC** [comment]
@Crispybuilt please look at the date of the video, it says 19.12.2009. the patch that was added just before that was 3.3.0a which was just a minor maintance at 14.12.2009. meaning this video should be according to the prenerf and original implemented content at patch 3.3.0.

Also DI punishes you by killing a player and removing 1 from game for 3min, from my experience if the first mark isn't delayed enough DI will expire, and the dead person has to be resurected during the fight in some way so it's not like it is without any consequences using it.

**Comment by StheNoble** [comment]
Ohai,

i am sorry to say it harsh, but your point is bullshit @Crispybuilt - in early descriptions of this encounter you can read that many guilds had to use DI during the first weeks of this encounter prenerf.

with the nerf to this fight and the removal of bloodpower generation of marks the reason for using DI was removed and also the prevention of bloodpower generation by shields like pw:s or similar was removed.

prenerf and after nerf it was always possible to use DI on any player including those with Marks. The general opinion though after nerf was that it is useless at that point to DI since it was just a bit effort for heals without Bloodpower generation.


it was never considered an exploit on retail and always possible to DI any player.

http://wowwiki.wikia.com/wiki/Patch_3.3.2_(undocumented_changes)
changes to the encounter regarding Shields (no mention of DI)


http://www.wowhead.com/npc=37813/deathbringer-saurfang#comments:id=936021

http://www.wowhead.com/npc=37813/deathbringer-saurfang#comments:id=945749

wowhead forum discussion from reatail abnout the usage of DI prenerf.
from the comments you can see that DI was usable prenerf and after nerf but wasnt considered any usefull anymore after the nerf:
http://www.wowhead.com/forums&topic=149412/saurfang-divine-intervention-worth-it


currently you created a situation with your "fix" that was never present on reatil and makes the fight much more difficult then it should be. before your "fix" it was allready a gatekeeper for most pugs and weeker guilds, but now it s much worse imo.



imo there are two options. have this fight prenerf sttus = bloodpower generation from marks + working DI + shields preventing BP genneration

OR

have it nerfed = marks generate no BP, DI still works, Shields/abosrbs dont prevent BP generation


dont try to make a mix or fiddle with game/class mechanics like DI just to make encounters harder.

**Comment by Lothloryen** [comment]
Hello there, it has been reverted and already deployed to live.

**Comment by StheNoble** [comment]
tyvm loth.

**Comment by ashido96** [comment]
Just to document because it's not readily apparent from posts. This is currently in complete pre-nerf status on live.

**Comment by lkrocek** [comment]
@Lothloryen I return back on dala-wow after more than half year and this bug is still up, can you deploy/fix it pls.
